src/wireframes/model/actions/loading.ts

```diff
@@ -37,7 +37,7 @@ export const loadDiagramFromServer =
 export const loadDiagramInternal =
```

Comment:
It is good to make adjustments and fix code formatting, but when the file has nothing to do with the code being edited, it is a little bit weird.

Reply:
Apologies, not exactly sure what happened here.